packages/web3torrent/tracker/package.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Maybe add a `script` that calls `packages/web3torrent/tracker/build-push-release.sh`

Reply:
3f8d70bb704af9c9eb26103b209ed484babb1e8b